cmd/root.go

```diff
@@ -354,6 +400,10 @@ func getFieldIDs(client jira.Client) error {
 	return nil
 }
 
+// compareIssues gets the list of GitHub issues updated since the `since` date,
+// gets the list of JIRA issues which have GitHub ID custom fields in that list,
+// then matches each one. If a JIRA issue already exists for a given GitHub issue,
+// it calls updateIssue; if no JIRA issue already exists, it calls createIssue.
```

Comment:
The comment should not details the implementation and functions used but rather the business logic, e.g. "If the issue already exists, it is updated; if no issue exists, a new one is created." Otherwise, everytime implementation changes slightly, this needs to be kept in sync.